app/src/main/java/fr/free/nrw/commons/Media.java

```diff
@@ -347,7 +347,14 @@ public void setDateCreated(Date date) {
      */
     public @Nullable
     Date getDateUploaded() {
-        return dateUploaded;
+      String date = CommonsDateUtil.getIso8601DateFormatShort().format(dateUploaded);
+      Date uploadDate = dateUploaded;
+      try {
+        uploadDate = CommonsDateUtil.getIso8601DateFormatShort().parse(date);
```

Comment:
Why not directly CommonsDateUtil.getIso8601DateFormatShort().parse(dateUploaded.getTimestamp())?

Reply:
There isn't getTimestamp() for util.Date. I can do parse(dateUploaded.toString()). I would still have to catch the exception in the method.

Reply:
hello @ashishkumar468 I think we can just return the dateUploaded like before. This a log of a test I did with the previous ;

![log2server](https://user-images.githubusercontent.com/10153512/78230002-0a6b1c00-74d1-11ea-929c-4cf543a1e70a.PNG)

Reply:
Actually yes, the places where this is used convert them to appropriate format themselves

Reply:
Can you revert the changes in this method then?